bibtex/bib/quantum.bib

```diff
@@ -28,6 +17,70 @@ @Article{BB14
   Url                      = {http://dx.doi.org/10.1016/j.tcs.2014.05.025}
 }
 
+@Article{BB84,
+  Title                    = {Quantum cryptography: Public key distribution and coin tossing},
+  Author                   = {Charles H. Bennett and Gilles Brassard},
+  Journal                  = {IEEE International Conference on Computers, Systems and Signal Processing},
+  Year                     = {1984},
+  Pages                    = {175--179},
+
+  Owner                    = {chris},
+  Timestamp                = {2015.07.24}
+}
+
+@Article{BBCS92,
```

Comment:
This should be InProceedings instead of Article, it's in the CRYPTO proceedings:
http://link.springer.com/chapter/10.1007%2F3-540-46766-1_29